include/uk/plat/lcpu.h

```diff
@@ -148,6 +148,9 @@ __u32 ukplat_lcpu_count(void);
  *   elements in lcpuidx, and [OUT] the number of successfully started CPUs in
  *   sequential order of lcpuidx. If the call succeeds, input and output values
  *   are equal. Must be NULL if lcpuidx is NULL
+ * @param pt_base array of pointers to the base of pagetables, on for each
+ *        logical CPU. If lcpuidx is NULL, must be ukplat_lcpu_count() - 1
```

Comment:
```suggestion
 *        logical CPU. If lcpuidx is NULL, must be ukplat_lcpu_count() - 1
```

I am not sure I can follow. Who should be  `ukplat_lcpu_count() - 1`?